### ISSUE TITLE
Installation Error

### ISSUE THREAD

**Issue by perugiapg12** [Anjok07/ultimatevocalremovergui#991]
Hi,
my pc
LMDE 6 Faye (linux mint dedian edition)
Cinnamon 5.8.4
Kernel Linux 6.1.0-13-amd64
CPU AMD Ryzen 7 5700U with Radeon Graphics × 8
RAM 37.0 GB
SSD Western Digital 1TB
GPU Advanced Micro Devices, Inc. [AMD/ATI] Lucienne

Problem 
error: externally-managed-environment

× This environment is externally managed
╰─> To install Python packages system-wide, try apt install
    python3-xyz, where xyz is the package you are trying to
    install.
    
    If you wish to install a non-Debian-packaged Python package,
    create a virtual environment using python3 -m venv path/to/venv.
    Then use path/to/venv/bin/python and path/to/venv/bin/pip. Make
    sure you have python3-full installed.
    
    If you wish to install a non-Debian packaged Python application,
    it may be easiest to use pipx install xyz, which will manage a
    virtual environment for you. Make sure you have pipx installed.
    
    See /usr/share/doc/python3.11/README.venv for more information.

note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
hint: See PEP 668 for the detailed specification.

bye from Italy

**Comment by mithusingh32** [comment]
Yes this is by design for Debian based installations. The README is a bit out of date. 

Check #1068 for the steps I took to install on Kubuntu
